src/test/test_sign.py

```diff
@@ -377,5 +382,49 @@ def add_by_final_vbf(a_hex, b_hex):
         self.assertEqual(mul(zero, scalar), (WALLY_OK, zero_hex))
         self.assertEqual(mul(zero, negative), (WALLY_OK, zero_hex))
 
+    def test_bip340_sigs(self):
+
+        made_sig = False
+
+        for case in self.get_bip340_sign_cases():
+            priv_key = make_cbuffer(case["priv_key"])
+            pub_key = make_cbuffer(case["pub_key"])
+            aux_rand = make_cbuffer(case["aux_rand"])
+            msg = make_cbuffer(case["msg"])
+            sig = make_cbuffer(case["sig"])
+            verification = case["verification"]
+
+            sig_out = self.cbufferize(['00' * 64])[0]
+
+            # We don't expose aux_rand, so only test default value for now
+            if priv_key[1] == 32 and aux_rand[0] == bytes(32):
```

Comment:
Please fix the existing typo `EX_PRIV_KEY_LEN` to `EC_PRIV_KEY_LEN` and use that in this test instead of `32` (the aux rand length is ok as 32 for now as we don't have a constant for it).

Reply:
done